### PR TITLE
Time dilation test harness

### PR DESCRIPTION
**Pull request by jonathathan**

### CODE REVIEW

**Comment by jonathathan** [comment]
@gdb @nottombrown this puts most of the stress test harness stuff in a new top-level `env_stress_tests/` dir. Any sort of review is welcome!

File: env_stress_tests/run/pod.yaml.template [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        echo -n | nc localhost 9999
+    "]
+    resources:
+      requests:
+        cpu: 2
+  - name: env
+    image: docker.openai.com/universe.flashgames
+    command: ["/bin/bash", "-ce", "
+        /app/universe-envs/flashgames/init run &
+        python -c 'import socket; s = socket.socket(); s.bind((\"localhost\", 9999)); s.listen(1); s.accept()[0].close()' # don't have netcat
```
Comment (gdb) [inline env_stress_tests/run/pod.yaml.template]: Should we just install netcat?

**Comment by gdb** [comment]
I think this code would be better off somewhere else, though I'm not sure where exactly. Perhaps a `universe-contrib` repo?

**Comment by jonathathan** [comment]
@gdb what other sorts of things might live in `universe-contrib`? 

I'm also not opposed to putting this in its own repo.

**Comment by gdb** [comment]
I think basically more utilities like this, which aren't directly supported
or tied to the Universe core functionality.


- gdb

**Comment by jonathathan** [comment]
Merged into new repo https://github.com/openai/universe-contrib

**Pull request closed without merging**